exporter/otlp/lib/opentelemetry/exporter/otlp/exporter.rb

```diff
@@ -190,6 +190,12 @@ def send_bytes(bytes, timeout:) # rubocop:disable Metrics/AbcSize, Metrics/Cyclo
           rescue SocketError
             retry if backoff?(retry_count: retry_count += 1, reason: 'socket_error')
             return FAILURE
+          rescue SystemCallError
+            retry if backoff?(retry_count: retry_count += 1, reason: 'system_call_error')
```

Comment:
Is there more detail we can provide here to distinguish different platform-dependent errors? The full `message` is obviously too verbose. Would `.class.name` be reasonable?

Reply:
Yes, that's a good idea